stitching/cli/stitch.py

```diff
@@ -294,6 +309,7 @@ def main():
     verbose_dir = args_dict.pop("verbose_dir")
     preview = args_dict.pop("preview")
     output = args_dict.pop("output")
+    output_params = args_dict.pop("output_params", [])
```

Comment:
You can remove the empty list here, since its already the default in the argument parser

Reply:
Done! (force pushed)